tests/test_application_summary.py

```diff
@@ -112,16 +113,19 @@
 
 
 def test_serialise_application_summary():
-    application_list = TEST_APPLICATION_STORE_DATA
+    with app.test_request_context():
+        application_list = TEST_APPLICATION_STORE_DATA
 
-    applications = [
-        ApplicationSummary.from_dict(application)
-        for application in application_list
-    ]
-    assert len(applications) == 3
-    assert applications[0].started_at.__class__.__name__ == "datetime"
-    assert str(applications[0].started_at.tzinfo) == "Europe/London"
-    assert applications[1].last_edited is None
+        applications = [
+            ApplicationSummary.from_dict(application)
+            for application in application_list
+        ]
+        assert len(applications) == 3
+        assert applications[0].started_at.__class__.__name__ == "datetime"
```

Comment:
I'd maybe use an instance check here instead of a string assertion, I don't think this is too big of a deal but in theory there _could_ be other classes from different packages named `datetime` - very unlikely but the instance check is just more standard practice.  

```suggestion
        assert assert isinstance(applications[0].started_at, datetime.datetime)
```